nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/services/impl/ContainerMapper.java

```diff
@@ -470,7 +470,7 @@ public void unregister(String contextRoot) {
 
     public void register(final Endpoint endpoint) {
         if (LOGGER.isLoggable(Level.FINE)) {
-            LOGGER.log(Level.FINE, "MAPPER({0}) REGISTER endpoint: {1}", endpoint);
+            LOGGER.log(Level.FINE, "MAPPER({0}) REGISTER endpoint: {1}", new Object[] { this, endpoint });
```

Comment:
Found a similar `log` call in the class which was using `this` as the "MAPPER" so it thought it should be same here.